### PR TITLE
[new release] nanosvg (0.2)

### PR DESCRIPTION
**Pull request by Armael**
Simple SVG parser and rasterizer

- Project page: <a href="https://github.com/Armael/ocaml-nanosvg">https://github.com/Armael/ocaml-nanosvg</a>

##### CHANGES:

- parse and decode XML entities in SVG text nodes

### CODE REVIEW

File: packages/nanosvg/nanosvg.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+maintainer: ["Armaël Guéneau <armael.gueneau@ens-lyon.org>"]
+authors: [
+  "Mikko Mononen memon@inside.org"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.org>"
+]
+license: "MIT"
+homepage: "https://github.com/Armael/ocaml-nanosvg"
+bug-reports: "https://github.com/Armael/ocaml-nanosvg/issues"
+depends: [
+  "ocaml" {>= "4.12.0"}
```
Comment (kit-ty-kate) [inline packages/nanosvg/nanosvg.0.2/opam:14]:
```suggestion
  "ocaml" {>= "4.14.0"}
```
It looks like you're using OCaml 4.14 features:
```
#=== ERROR while compiling nanosvg.0.2 ========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.13.1 | pinned(https://github.com/Armael/ocaml-nanosvg/releases/download/0.2/nanosvg-0.2.tbz)
# path                 ~/.opam/4.13/.opam-switch/build/nanosvg.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p nanosvg -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/nanosvg-7-ab9860.env
# output-file          ~/.opam/log/nanosvg-7-ab9860.out
### output ###
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.nanosvg.objs/byte -intf-suffix .ml -no-alias-deps -o lib/.nanosvg.objs/byte/nanosvg.cmo -c -impl lib/nanosvg.ml)
# File "lib/nanosvg.ml", line 355, characters 18-40:
# 355 |         let dec = String.get_utf_8_uchar txt pos in
#                         ^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value String.get_utf_8_uchar
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlopt.opt -w -40 -g -I lib/.nanosvg.objs/byte -I lib/.nanosvg.objs/native -intf-suffix .ml -no-alias-deps -o lib/.nanosvg.objs/native/nanosvg.cmx -c -impl lib/nanosvg.ml)
# File "lib/nanosvg.ml", line 355, characters 18-40:
# 355 |         let dec = String.get_utf_8_uchar txt pos in
#                         ^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value String.get_utf_8_uchar
```

Comment (Armael) [reply packages/nanosvg/nanosvg.0.2/opam]: damn, indeed. thanks.

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by kit-ty-kate** [comment]
Could you return the fix in the opam file upstream so that it can be taken into account for next time?

**Pull request merged**